docs/.vuepress/public/presets/toml/tokyo-night.toml

```
@@ -1,64 +1,205 @@
 format = """
-[░▒▓](#a3aed2)\
-[  ](bg:#a3aed2 fg:#090c0c)\
-[](bg:#769ff0 fg:#a3aed2)\
+[ ](bg:blurple_darkest)\
+[ ](bg:blurple_dark)\
+[ ](bg:blurple_darker)\
+[ ](bg:blurple)\
+$username\
+$os\
+[](bg:blue fg:blurple)\
 $directory\
-[](fg:#769ff0 bg:#394260)\
+[](fg:blue bg:blurple_dark)\
 $git_branch\
 $git_status\
-[](fg:#394260 bg:#212736)\
+[](fg:blurple_dark bg:blurple_darkest)\
+$java\
 $nodejs\
+$python\
 $rust\
 $golang\
 $php\
-[](fg:#212736 bg:#1d2230)\
+$c\
+$elixir\
+$elm\
+$gradle\
+$haskell\
+$julia\
+$nim\
+$scala\
+[](fg:blurple_darkest bg:black)\
 $time\
-[ ](fg:#1d2230)\
+[ ](fg:black)\
 \n$character"""
```

Sorry, I totally for got the following suggestion
```suggestion
$line_break$character"""
```

The `[line_break]` module, is enabled by default. By using it, the user can disable it easily with `starship config line_break.disabled false`, if they so desire it.